### PR TITLE
Tests us1 review

### PR DESCRIPTION
**Pull request by sjoerdbr**

### CODE REVIEW

File: src/app/components/module-component/module.component.spec.ts [hunk 1]
```diff
@@ -103,6 +103,6 @@ describe('Testing module component', () => {
     el = fixture.debugElement.query(By.css('.skill-matrix')).nativeElement;
     // have to check if it exists n has some values.
     // expect(el.).toBe(1);
-  });
+  });*/
```
Comment (Lehren) [inline src/app/components/module-component/module.component.spec.ts:106]: Why is this test commented?

File: src/app/components/module-overview-component/module-overview.component.spec.ts [hunk 2]
```diff
@@ -102,5 +102,5 @@ describe('ModuleOverviewComponent', () => {
     // get the first module element
     el = fixture.debugElement.query(By.css('h4')).nativeElement;
     expect(el.innerText).toBe('JAV1');
-  });
+  });*/
```
Comment (Lehren) [inline src/app/components/module-overview-component/module-overview.component.spec.ts]: And this one

File: src/app/components/skillmatrix-component/skillmatrix.component.spec.ts [hunk 3]
```diff
@@ -1,93 +1,62 @@
...
-        observer.next(modulemodel);
-        observer.complete();
-      });
-    }
-  };
-
+  let component: TestHostComponent;
+  let testhostfixture: ComponentFixture<TestHostComponent>;
+  // let de:      DebugElement;
+  // let el:      HTMLElement;
```
Comment (Lehren) [inline src/app/components/skillmatrix-component/skillmatrix.component.spec.ts]: More commented code

**Review by Lehren — DISMISSED** [review]

File: src/app/components/skillmatrix-component/skillmatrix.component.spec.ts [hunk 4]
```diff
@@ -1,93 +1,62 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
-import {DebugElement} from '@angular/core';
+import {Component, DebugElement} from '@angular/core';
 import {Observable} from 'rxjs/Observable';
```
Comment (NNieuwenhuis) [inline src/app/components/skillmatrix-component/skillmatrix.component.spec.ts]: unused import

File: src/app/components/skillmatrix-component/skillmatrix.component.spec.ts [hunk 5]
```diff
@@ -1,93 +1,62 @@
...
 import {ModuleContent} from '../../models/modulecontent.model';
```
Comment (NNieuwenhuis) [inline src/app/components/skillmatrix-component/skillmatrix.component.spec.ts]: unused import

File: src/app/components/skillmatrix-component/skillmatrix.component.spec.ts [hunk 6]
```diff
@@ -1,93 +1,62 @@
...
 import {Subscriber} from 'rxjs/Subscriber';
```
Comment (NNieuwenhuis) [inline src/app/components/skillmatrix-component/skillmatrix.component.spec.ts]: unused import

File: src/app/components/skillmatrix-component/skillmatrix.component.spec.ts [hunk 7]
```diff
@@ -1,93 +1,62 @@
...
 import {BackendService} from '../../backend.service';
```
Comment (NNieuwenhuis) [inline src/app/components/skillmatrix-component/skillmatrix.component.spec.ts]: unused import

**Review by NNieuwenhuis — CHANGES_REQUESTED** [review]
Do you need the comments later? Otherwise delete them please.

File: src/app/components/module-component/module.component.spec.ts (hunk 1 above)
Comment (sjoerdbr) [reply src/app/components/module-component/module.component.spec.ts:106]: For reference purposes

**Review by NNieuwenhuis — APPROVED** [review]

**Pull request merged**